aws_lambda_powertools/utilities/parameters/dynamodb.py

```diff
@@ -209,4 +204,4 @@ def _get_multiple(self, path: str, **sdk_options) -> Dict[str, str]:
             response = self.table.query(**sdk_options)
             items.extend(response.get("Items", []))
 
-        return {item[self.sort_attr]: item[self.value_attr] for item in items}
+        return {str(item[self.sort_attr]): str(item[self.value_attr]) for item in items}
```

Comment:
why always str? dynamodb can store all sorts of data types

Reply:
@risenberg-cyberark  the outer typing is excepting a `Dict[str, str]`

Reply:
So, if this is practically true for parameters, we should then update the outer type to `Dict[str, Any]`, but i am not sure this would work correctly with the right of the parameters features.

Reply:
@heitorlessa do you think this type casting is the right idea? Or should we ignore these kind of Mypy issues?